sklearn/cross_validation.py

```diff
@@ -300,7 +307,7 @@ class KFold(_BaseKFold):
 
     See also
     --------
-    StratifiedKFold: take label information into account to avoid building
+    StratifiedKFold take label information into account to avoid building
```

Comment:
This seems to render properly here - http://rvraghav93.github.io/doc_builds/modules/generated/sklearn.model_selection.KFold.html#sklearn.model_selection.KFold